### PR TITLE
Odometry visualization

### PR DESCRIPTION
**Pull request by Josecisneros001**
Test Rviz Model, Visualize Odometry according to ticks of encoders simulated.
#42 

Rviz model for Base control
[X] Investigation
[X] Prototype.
! Deliverable: PR with Rviz model and documentation. (There is a new entry to the wiki in Navigation/base_control)

### CODE REVIEW

File: catkin_home/src/navigation/base_control/rviz/base_control.rviz [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
...fb000000120056006900650077007300200054006f006f02000001df000002110000018500000122fb000000200054006f006f006c002000500072006f0070006500720074006900650073003203000002880000011d000002210000017afb000000100044006900730070006c006100790073010000003d00000363000000c900fffffffb0000002000730065006c0065006300740069006f006e00200062007500660066006500720200000138000000aa0000023a00000294fb00000014005700690064006500530074006500720065006f02000000e6000000d2000003ee0000030bfb0000000c004b0069006e0065006300740200000186000001060000030c00000261000000010000010f00000306fc0200000003fb0000001e0054006f006f006c002000500072006f00700065007200740069006500730100000041000000780000000000000000fb0000000a00560069006500770073000000003d00000306000000a400fffffffb0000001200530065006c0065006300740069006f006e010000025a000000b200000000000000000000000200000490000000a9fc0100000001fb0000000a00560069006500770073030000004e00000080000002e100000197000000030000073d0000003efc0100000002fb0000000800540069006d006501000000000000073d000002eb00fffffffb0000000800540069006d00650100000000000004500000000000000000000005e10000036300000004000000040000000800000008fc0000000100000002000000010000000a0054006f006f006c00730100000000ffffffff0000000000000000
```
Comment (paulvazbad) [inline catkin_home/src/navigation/base_control/rviz/base_control.rviz:140]: What is the meaning of this? Just curious.

File: catkin_home/src/navigation/base_control/scripts/encoders.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    encFrontL  =    [forward ,backward,forward,backward,forward,backward,stop   ,stop    ,backward,forward ]
+    encBackL   =    [backward,forward ,forward,backward,stop   ,stop    ,forward,backward,backward,forward ]
+    encFrontR  =    [backward,forward ,forward,backward,stop   ,stop    ,forward,backward,forward ,backward]
+    encBackR   =    [forward ,backward,forward,backward,forward,backward,stop   ,stop    ,forward ,backward]
+    
+    t0 = rospy.Time.now().to_sec()
+    t1 = rospy.Time.now().to_sec()
+    indexDirection = 0
+    while not rospy.is_shutdown():
+
```
Comment (paulvazbad) [inline catkin_home/src/navigation/base_control/scripts/encoders.py:35]: Extra space

File: catkin_home/src/navigation/base_control/scripts/encoders.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            t0 = rospy.Time.now().to_sec()
+            
+            front_publisher.publish(enc_msgFront)
+            back_publisher.publish(enc_msgBack)
+        
+        if rospy.Time.now().to_sec() - t1 > 2.5:
+            t1 = rospy.Time.now().to_sec()
+            indexDirection = indexDirection + 1
+            if indexDirection > 9:
+                indexDirection = 0
```
Comment (paulvazbad) [inline catkin_home/src/navigation/base_control/scripts/encoders.py:46]: indexDirection = (indexDirection +1) % 10

**Review by paulvazbad — APPROVED** [review]

File: catkin_home/src/navigation/base_control/rviz/base_control.rviz (hunk 1 above)
Comment (Josecisneros001) [reply catkin_home/src/navigation/base_control/rviz/base_control.rviz:140]: It is just the configuration file of the environment of rviz, this is passed as parameter in the launch file in rviz node.

**Pull request merged**